scripts/triggers-end-to-end-tests/tests.ts

```diff
@@ -261,39 +261,114 @@ describe("storage emulator function triggers", () => {
     expect(response.status).to.equal(200);
     await new Promise((resolve) => setTimeout(resolve, EMULATORS_WRITE_DELAY_MS));
   });
 
   it("should have triggered cloud functions", () => {
-    // on object create two events fire (finalize & metadata update)
+    /* on object create two events fire (finalize & metadata update) */
+    // default bucket
     expect(test.storageFinalizedTriggerCount).to.equal(1);
     expect(test.storageMetadataTriggerCount).to.equal(1);
     expect(test.storageV2FinalizedTriggerCount).to.equal(1);
     expect(test.storageV2MetadataTriggerCount).to.equal(1);
+    expect(test.storageDeletedTriggerCount).to.equal(0);
+    expect(test.storageV2DeletedTriggerCount).to.equal(0);
+    // specific bucket
+    expect(test.storageBucketFinalizedTriggerCount).to.equal(0);
+    expect(test.storageBucketMetadataTriggerCount).to.equal(0);
+    expect(test.storageBucketV2FinalizedTriggerCount).to.equal(0);
+    expect(test.storageBucketV2MetadataTriggerCount).to.equal(0);
+    expect(test.storageBucketDeletedTriggerCount).to.equal(0);
+    expect(test.storageBucketV2DeletedTriggerCount).to.equal(0);
+    test.resetCounts();
   });
 
-  it("should write, update, and delete from the storage emulator", async function (this) {
+  it("should write to a specific bucket of storage emulator", async function (this) {
     this.timeout(EMULATOR_TEST_TIMEOUT);
 
-    const response = await test.updateDeleteFromStorage();
+    const response = await test.writeToSpecificStorageBucket();
     expect(response.status).to.equal(200);
     await new Promise((resolve) => setTimeout(resolve, EMULATORS_WRITE_DELAY_MS));
   });
 
   it("should have triggered cloud functions", () => {
-    // on update two events fire (finalize & metadata update)
-    expect(test.storageFinalizedTriggerCount).to.equal(2);
-    expect(test.storageMetadataTriggerCount).to.equal(2);
-    expect(test.storageV2FinalizedTriggerCount).to.equal(2);
-    expect(test.storageV2MetadataTriggerCount).to.equal(2);
-    // on delete one event fires (delete)
+    /* on object create two events fire (finalize & metadata update) */
+    // default bucket
+    expect(test.storageFinalizedTriggerCount).to.equal(0);
+    expect(test.storageMetadataTriggerCount).to.equal(0);
+    expect(test.storageV2FinalizedTriggerCount).to.equal(0);
+    expect(test.storageV2MetadataTriggerCount).to.equal(0);
+    expect(test.storageDeletedTriggerCount).to.equal(0);
+    expect(test.storageV2DeletedTriggerCount).to.equal(0);
+    // specific bucket
+    expect(test.storageBucketFinalizedTriggerCount).to.equal(1);
+    expect(test.storageBucketMetadataTriggerCount).to.equal(1);
+    expect(test.storageBucketV2FinalizedTriggerCount).to.equal(1);
+    expect(test.storageBucketV2MetadataTriggerCount).to.equal(1);
+    expect(test.storageBucketDeletedTriggerCount).to.equal(0);
+    expect(test.storageBucketV2DeletedTriggerCount).to.equal(0);
+    test.resetCounts();
```

Comment:
aside* I'd like to see this function be called in `afterEach` hook, but I see that this isn't possible due to the way the test cases are structured today :/.